GOVERNANCE.md

```diff
@@ -67,4 +67,5 @@ For example:
 
 > [runtime-spec adopted]: Tag 0647920 as 1.0.0-rc (+6 -0 #3)
 
+
```

Comment:
I'd rather not churn this file unless we need to.  Can we drop this insertion?  Or if you feel particularly strongly about it, insert it [upstream][1]?

[1]: https://github.com/opencontainers/project-template